### PR TITLE
DOC: translation for the contributing doc

### PR DESCRIPTION
**Pull request by RayJi01**
## What do these changes do?
Translate the document "Contributing Xorbits"

### CODE REVIEW

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SOME DESCRIPTIVE TITLE.
+   # SOME DESCRIPTIVE TITLE.
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:1]: Why adding whitespaces?

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 2]
```diff
@@ -302,143 +302,148 @@ msgid ""
 " created the branch, check the section on :ref:`updating a PR "
 "<contributing.update-pr>`."
 msgstr ""
-"当你想添加主干分支的改动到你的功能分支时，请见 :ref:`更新 PR <contributing.update-pr>` 。"
+"当你想添加主干分支的改动到你的功能分支时，请见 :ref:`更新你的pull request <contributing.update-pr>` 。"
 
 #: ../../source/development/contributing.rst:172
 msgid "Contributing your changes to Xorbits"
-msgstr ""
+msgstr "将你的更改成果同步给Xorbits"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:309]:
```suggestion
msgstr "将改动贡献到 Xorbits"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 3]
```diff
@@ -302,143 +302,148 @@ msgid ""
...
 msgid "Committing your code"
-msgstr ""
+msgstr "提交你的代码"
 
 #: ../../source/development/contributing.rst:179
 msgid ""
 "Keep style fixes to a separate commit to make your pull request more "
 "readable."
-msgstr ""
+msgstr "请保持每次commit的代码风格一致，以便使你的pull request更具有可读性"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:319]:
```suggestion
msgstr "请保持每次commit的代码风格一致，以便使你的pull request更具有可读性。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 4]
```diff
@@ -302,143 +302,148 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:202
 msgid "BUG: Bug fix"
-msgstr ""
+msgstr "BUG: Bug 修复"
 
 #: ../../source/development/contributing.rst:203
 msgid "DOC: Additions/updates to documentation"
-msgstr ""
+msgstr "DOC: 文件(documentation)的删改与添加"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:358]:
```suggestion
msgstr "DOC: 文档(documentation)的删改与添加"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 5]
```diff
@@ -447,31 +452,33 @@ msgid ""
...
 msgid ""
 "Navigate to your repository on GitHub -- https://github.com/your-user-"
 "name/xorbits"
-msgstr ""
+msgstr "跳转至你的Github repository -- https://github.com/your-user-name/xorbits"
 
 #: ../../source/development/contributing.rst:259
 msgid "Click on ``Branches``"
-msgstr ""
+msgstr "点击 ``Compare``"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:467]: Branch?

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 6]
```diff
@@ -481,39 +488,40 @@ msgid ""
 " version.  This pull request and its associated changes will eventually "
 "be committed to the main branch and available in the next release.  To "
 "submit a pull request:"
-msgstr ""
+msgstr "若无异常，你就准备好提交pull request了。Pull Request是将本地仓库中的代码提供给 GitHub 社区并进行查看和最终合并到主要版本的方式"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:491]:
```suggestion
msgstr "若无异常，你就准备好提交pull request了。Pull Request是将本地仓库中的代码提供给 GitHub 社区并进行查看和最终合并到主要版本的方式。"
```
Don't forget to add punctuations.

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 7]
```diff
@@ -481,39 +488,40 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:277
 msgid "Write a description of your changes in the ``Preview Discussion`` tab"
-msgstr ""
+msgstr "在 ``Preview Discussion`` 中描述你的更改"
 
 #: ../../source/development/contributing.rst:278
 msgid "Click ``Send Pull Request``."
-msgstr ""
+msgstr "点击 ``Send Pull Request``"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:514]: Ditto.

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 8]
```diff
@@ -481,39 +488,40 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:280
 msgid ""
 "This request then goes to the repository maintainers, and they will "
 "review the code."
-msgstr ""
+msgstr "你的请求将会送达到核心开发人员端并被审查"
```
Comment (qianduoduo0904) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:520]: Ditto.

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po (hunk 5 above)
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:467]:
```suggestion
msgstr "点击 ``Branches``"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po (hunk 7 above)
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:514]:
```suggestion
msgstr "点击 ``Send Pull Request``。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po (hunk 8 above)
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:520]:
```suggestion
msgstr "你的请求将会送达到核心开发人员端并被审查。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po (hunk 1 above)
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:1]:
```suggestion
# SOME DESCRIPTIVE TITLE.
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po (hunk 5 above)
Comment (RayJi01) [reply doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po]: changed in the new version

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 9]
```diff
@@ -522,31 +530,32 @@ msgid ""
 " branch, add a new commit to that branch, push it to GitHub, and the pull"
 " request will be automatically updated.  Pushing them to GitHub again is "
 "done by::"
-msgstr ""
+msgstr "基于核心开发人员对你的代码的审查结果，你可能需要对其做一些改动。你可以在你的分支上进行进行修改，随后commit并将这些修改"
+"提交到Github上。你的pull request会根据提交结果自动更新。用以下操作将你的更新提交至Github::"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:534]:
```suggestion
"提交到Github上。你的 pull request 会根据提交结果自动更新。用以下操作将你的更新提交至Github::"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 10]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:295
 msgid ""
 "This will automatically update your pull request with the latest code and"
 " restart the :any:`Continuous Integration <contributing.ci>` tests."
-msgstr ""
+msgstr "这串代码会自动更新你的pull request至最新的代码版本并且重新执行 :any:`Continuous Integration <contributing.ci>` 测试"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:540]:
```suggestion
msgstr "这串代码会自动更新你的 pull request 至最新的代码版本并且重新执行 :any:`Continuous Integration <contributing.ci>` 测试"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 11]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:298
 msgid ""
 "Another reason you might need to update your pull request is to solve "
 "conflicts with changes that have been merged into the main branch since "
 "you opened your pull request."
-msgstr ""
+msgstr "另外， 当与自你提交pull request以来已合并到主分支的更改产生的冲突时，你也需要对pull request进行更改。"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:547]:
```suggestion
msgstr "另外， 当与自你提交 pull request 以来已合并到主分支的更改产生的冲突时，你也需要对pull request进行更改。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 12]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
-msgstr ""
+msgstr "你需要在你的分支上执行 \"merge upstream main\" 来解决上述问题"
 
 #: ../../source/development/contributing.rst:308
 msgid ""
 "If there are no conflicts (or they could be fixed automatically), a file "
 "with a default commit message will open, and you can simply save and quit"
 " this file."
-msgstr ""
+msgstr "如果没有冲突，将会自动打开一份含有默认commit message的文件，然后你就可以保存并关闭这个文档。"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:558]:
```suggestion
msgstr "如果没有冲突，将会自动打开一份含有默认 commit message 的文件，然后你就可以保存并关闭这个文档。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 13]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
+msgstr "如果你在要更新同步本地分支与主分支的代码之前有本地分支还有未提交的更改，你需要在更新之前将它们存储(stash)起来"
+"(请参见文档 `stash docs <https://git-scm.com/book/en/v2/Git-Tools-Stashing-and-"
+"Cleaning>`__). 这将有效地保存你的更改，并且在更新后可以重新应用它们。"
 
 #: ../../source/development/contributing.rst:322
 msgid ""
 "After the feature branch has been update locally, you can now update your"
 " pull request by pushing to the branch on GitHub::"
-msgstr ""
+msgstr "在本地更新了功能分支之后，你可以通过将其推送(push)到 GitHub 上的分支来更新你的pull request"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:585]:
```suggestion
msgstr "在本地更新了功能分支之后，你可以通过将其推送到 GitHub 上的分支来更新你的 pull request"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 14]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 msgid "Autofixing formatting errors"
-msgstr ""
+msgstr "自动修正格式问题"
 
 #: ../../source/development/contributing.rst:330
 msgid ""
 "We use several styling checks (e.g. ``black``, ``flake8``, ``isort``) "
 "which are run after you make a pull request."
-msgstr ""
+msgstr "Xorbits项目使用多个样式检查工具（例如 ``black``、 ``flake8``、 ``isort``），这些工具会在你发起pull request后运行。"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:595]:
```suggestion
msgstr "Xorbits项目使用多个样式检查工具（例如 ``black``、 ``flake8``、 ``isort``），这些工具会在你发起 pull request 后运行。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 15]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
+msgstr "(可选) 删除已合并的分支。"
 
 #: ../../source/development/contributing.rst:340
 msgid ""
 "Once your feature branch is accepted into upstream, you'll probably want "
 "to get rid of the branch. First, merge upstream main into your branch so "
 "git knows it is safe to delete your branch::"
-msgstr ""
+msgstr "当upstream 接受并合并了你的功能分支后，你可以删掉这些功能分支。"
+"首先，将upstream main分支合并到你的分支，让git了解到你可以安全的删除当前功能分支"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:617]:
```suggestion
"首先，将 upstream main 分支合并到你的分支，让git了解到你可以安全的删除当前功能分支"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 16]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 msgid "Then you can do::"
-msgstr ""
+msgstr "然后你可以执行::"
 
 #: ../../source/development/contributing.rst:352
 msgid ""
 "Make sure you use a lower-case ``-d``, or else git won't warn you if your"
 " feature branch has not actually been merged."
-msgstr ""
+msgstr "确保你输入的是小写 ``-d`` ，否则若你的功能分支没有合并成功，git将不会警告。"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:627]:
```suggestion
msgstr "确保你输入的是小写 ``-d`` ，否则若你的功能分支没有合并成功，git 将不会警告。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 17]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:355
 msgid "The branch will still exist on GitHub, so to delete it there do::"
-msgstr ""
+msgstr "当你执行完上述操作以后，这个分支还会存在在Github上，你可以通过以下指令进行删除::"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:631]:
```suggestion
msgstr "当你执行完上述操作以后，这个分支还会存在在 GitHub 上，你可以通过以下指令进行删除::"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 18]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 "people are responsible for reviewing all of the contributions, which can "
 "often lead to bottlenecks."
-msgstr ""
+msgstr "如果您已经进入了 `检查你的代码`_ 阶段，核心贡献者之一可能会进行审查。"
+"然而，只有少数人负责审查所有的贡献，这往往会导致我们的审查无法顾及所有的需求。"
 
 #: ../../source/development/contributing.rst:367
 msgid "To improve the chances of your pull request being reviewed, you should:"
-msgstr ""
+msgstr "为了提高你的pull request被核心开发者查看的机会，你应该:"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:648]:
```suggestion
msgstr "为了提高你的 pull request 被核心开发者查看的机会，你应该:"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 19]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:369
 msgid ""
 "**Reference an open issue** for non-trivial changes to clarify the PR's "
 "purpose"
-msgstr ""
+msgstr "对于较为复杂的更改，可以 **引用一个开放的问题(open issue)** 来明确pull request的目的。"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:654]:
```suggestion
msgstr "对于较为复杂的更改，可以 **引用一个开放的问题(open issue)** 来明确 pull request 的目的。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 20]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:370
 msgid ""
 "**Ensure you have appropriate tests**. These should be the first part of "
 "any PR"
-msgstr ""
+msgstr "**恰当全面的测试** 是所有PR提交的首要部分。"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:660]:
```suggestion
msgstr "**恰当全面的测试** 是所有 pull request 提交的首要部分。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 21]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:371
 msgid ""
 "**Keep your pull requests as simple as possible**. Larger PRs take longer"
 " to review"
-msgstr ""
+msgstr "**PR 应当越简洁越好** ，因为越长的PRs需要越久的审查时间。 "
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:666]:
```suggestion
msgstr "**pull request 应当越简洁越好** ，因为越长的PRs需要越久的审查时间。 "
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 22]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:372
 msgid ""
 "**Ensure that CI is in a green state**. Reviewers may not even look "
 "otherwise"
-msgstr ""
+msgstr "审查人可能不会审查不通过 **Github Continuous Integration check** 的PRs，所以请确保你的PRs提交时 **CI check** 为绿色(passed)"
```
Comment (RayJi01) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:672]:
```suggestion
msgstr "审查人可能不会审查不通过 **GitHub Continuous Integration check** 的 pull request，所以请确保你的 pull request 提交时 **CI check** 为绿色"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 23]
```diff
@@ -522,31 +530,32 @@ msgid ""
 " branch, add a new commit to that branch, push it to GitHub, and the pull"
 " request will be automatically updated.  Pushing them to GitHub again is "
 "done by::"
-msgstr ""
+msgstr "基于核心开发人员对你的代码的审查结果，你可能需要对其做一些改动。你可以在你的分支上进行进行修改，随后commit并将这些修改"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:533]:
```suggestion
msgstr "基于核心开发人员对你的代码的审查结果，你可能需要对其做一些改动。你可以在你的分支上进行进行修改，随后 commit 并将这些修改"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 24]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
+"提交到Github上。你的 pull request 会根据提交结果自动更新。用以下操作将你的更新提交至Github::"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:534]:
```suggestion
"提交至 GitHub 上。你的 pull request 会根据提交结果自动更新。用以下操作将你的更新提交至 GitHub::"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 25]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
-msgstr ""
+msgstr "这串代码会自动更新你的 pull request 至最新的代码版本并且重新执行 :any:`Continuous Integration <contributing.ci>` 测试"
 
 #: ../../source/development/contributing.rst:298
 msgid ""
 "Another reason you might need to update your pull request is to solve "
 "conflicts with changes that have been merged into the main branch since "
 "you opened your pull request."
-msgstr ""
+msgstr "另外， 当与自你提交 pull request 以来已合并到主分支的更改产生的冲突时，你也需要对pull request进行更改。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:547]:
```suggestion
msgstr "另外， 当与自你提交 pull request 以来已合并到主分支的更改产生的冲突时，你也需要对 pull request 进行更改。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 26]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:302
 msgid "To do this, you need to \"merge upstream main\" in your branch::"
-msgstr ""
+msgstr "你需要在你的分支上执行 \"merge upstream main\" 来解决上述问题"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:551]:
```suggestion
msgstr "你需要合并上游代码仓库的主干分支来解决上述问题::"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 27]
```diff
@@ -522,31 +530,32 @@ msgid ""
...
-msgstr ""
+msgstr "基于核心开发人员对你的代码的审查结果，你可能需要对其做一些改动。你可以在你的分支上进行进行修改，随后commit并将这些修改"
+"提交到Github上。你的 pull request 会根据提交结果自动更新。用以下操作将你的更新提交至Github::"
 
 #: ../../source/development/contributing.rst:295
 msgid ""
 "This will automatically update your pull request with the latest code and"
 " restart the :any:`Continuous Integration <contributing.ci>` tests."
-msgstr ""
+msgstr "这串代码会自动更新你的 pull request 至最新的代码版本并且重新执行 :any:`Continuous Integration <contributing.ci>` 测试"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:540]:
```suggestion
msgstr "这会自动更新你的 pull request 至最新的代码版本并且重新执行 :any:`持续集成 <contributing.ci>` 测试。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 28]
```diff
@@ -555,7 +564,8 @@ msgid ""
 "using-the-command-line/ for an explanation on how to do this. Once the "
 "conflicts are merged and the files where the conflicts were solved are "
 "added, you can run ``git commit`` to save those fixes."
-msgstr ""
+msgstr "如果有合并冲突，则你需要解决这些冲突，请通过以下例子：https://help.github.com/articles/resolving-a-merge-conflict-"
+"using-the-command-line/了解如何解决。当所有的冲突都被解决并合并后，执行 ``git commit`` 来保存这些更改。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:568]:
```suggestion
"using-the-command-line/ 了解如何解决。当所有的冲突都被解决并合并后，执行 ``git commit`` 来保存这些更改。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 29]
```diff
@@ -564,100 +574,107 @@ msgid ""
 "the `stash docs <https://git-scm.com/book/en/v2/Git-Tools-Stashing-and-"
 "Cleaning>`__). This will effectively store your changes and they can be "
 "reapplied after updating."
-msgstr ""
+msgstr "如果你在要更新同步本地分支与主分支的代码之前有本地分支还有未提交的更改，你需要在更新之前将它们存储(stash)起来"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:577]:
```suggestion
msgstr "如果你在要更新同步本地分支与主分支的代码之前有本地分支还有未提交的更改，你需要在更新之前将它们存储 (stash) 起来"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 30]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
+"(请参见文档 `stash docs <https://git-scm.com/book/en/v2/Git-Tools-Stashing-and-"
+"Cleaning>`__). 这将有效地保存你的更改，并且在更新后可以重新应用它们。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:579]:
```suggestion
"Cleaning>`__)。 这将有效地保存你的更改，并且在更新后可以重新应用它们。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 31]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 msgid "Autofixing formatting errors"
-msgstr ""
+msgstr "自动修正格式问题"
 
 #: ../../source/development/contributing.rst:330
 msgid ""
 "We use several styling checks (e.g. ``black``, ``flake8``, ``isort``) "
 "which are run after you make a pull request."
-msgstr ""
+msgstr "Xorbits项目使用多个样式检查工具（例如 ``black``、 ``flake8``、 ``isort``），这些工具会在你发起 pull request 后运行。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:595]:
```suggestion
msgstr "Xorbits 项目使用多个样式检查工具（例如 ``black``、 ``flake8``、 ``isort``），这些工具会在你发起 pull request 后运行。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 32]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:333
 msgid ""
 "To automatically fix formatting errors on each commit you make, you can "
 "set up pre-commit yourself. First, create a Python :ref:`environment "
 "<contributing_environment>` and then set up :ref:`pre-commit "
 "<contributing.pre-commit>`."
-msgstr ""
+msgstr "你可以通过自己配置pre-commit工具来自动修正每一次commit中的格式错误。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:603]:
```suggestion
msgstr "你可以通过自己配置 pre-commit 工具来自动修正每一次 commit 中的格式错误。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 33]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
-msgstr ""
+msgstr "(可选) 删除已合并的分支。"
 
 #: ../../source/development/contributing.rst:340
 msgid ""
 "Once your feature branch is accepted into upstream, you'll probably want "
 "to get rid of the branch. First, merge upstream main into your branch so "
 "git knows it is safe to delete your branch::"
-msgstr ""
+msgstr "当upstream 接受并合并了你的功能分支后，你可以删掉这些功能分支。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:616]:
```suggestion
msgstr "当上游代码仓库接受并合并了你的功能分支后，你可以删掉这些功能分支。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 34]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
+"首先，将 upstream main 分支合并到你的分支，让 git 了解到你可以安全的删除当前功能分支"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:617]:
```suggestion
"首先，将上游主干分支合并到你的分支，让 git 了解可以安全的删除当前功能分支::"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 35]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 
 #: ../../source/development/contributing.rst:363
 msgid ""
 "If you have made it to the `Review your code`_ phase, one of the core "
 "contributors may take a look. Please note however that a handful of "
 "people are responsible for reviewing all of the contributions, which can "
 "often lead to bottlenecks."
-msgstr ""
+msgstr "如果您已经进入了 `检查你的代码`_ 阶段，核心贡献者之一可能会进行审查。"
+"然而，只有少数人负责审查所有的贡献，这往往会导致我们的审查无法顾及所有的需求。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:644]:
```suggestion
"然而，少数核心开发者需要负责审查所有的贡献，这往往会导致我们的审查无法顾及所有的需求。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 36]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 msgid "To improve the chances of your pull request being reviewed, you should:"
-msgstr ""
+msgstr "为了提高你的 pull request 被核心开发者查看的机会，你应该:"
 
 #: ../../source/development/contributing.rst:369
 msgid ""
 "**Reference an open issue** for non-trivial changes to clarify the PR's "
 "purpose"
-msgstr ""
+msgstr "对于较为复杂的更改，可以 **引用一个开放的问题(open issue)** 来明确 pull request 的目的。"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:654]:
```suggestion
msgstr "对于较为复杂的更改，可以 **引用一个尚未解决的 issue** 来明确 pull request 的目的。"
```

File: doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po [hunk 37]
```diff
@@ -564,100 +574,107 @@ msgid ""
...
 " to review"
-msgstr ""
+msgstr "**pull request 应当越简洁越好** ，因为越长的 pull request 需要越久的审查时间。 "
 
 #: ../../source/development/contributing.rst:372
 msgid ""
 "**Ensure that CI is in a green state**. Reviewers may not even look "
 "otherwise"
-msgstr ""
+msgstr "审查人可能不会审查不通过 **GitHub Continuous Integration check** 的 pull request，所以请确保你的 pull request 提交时 **CI check** 为绿色"
```
Comment (UranusSeven) [inline doc/source/locale/zh_CN/LC_MESSAGES/development/contributing.po:672]:
```suggestion
msgstr "审查人可能不会审查不通过 **GitHub 持续集成测试** 的 pull request，所以请确保你的 pull request 提交时 **CI check** 为绿色"
```

**Review by UranusSeven — APPROVED** [review]
LGTM.

**Review by qianduoduo0904 — APPROVED** [review]
LGTM

**Pull request merged**